### DISCUSSION TITLE
Contact Team Members if vulnerabilities found

### DISCUSSION THREAD

**Discussion by JoergBruenner** [DependencyTrack/dependency-track#2510] [Q&A]
Hi all,

I search my BOM base to find vulnerable projects by API. If I found one I'd like to contact the owner by mail. 
I'm using teams to group managed users. The teams are matched to projects by the portfolio access control feature.

The only way I've seen to solve my problem is:
1. get all Teams and their UUIDs by the "Get All Teams"-API call (/v1/team)
2. put the TeamUUID in the "Get ACL"-API call (/v1/acl/team/{uuid})
3. find the vulnerable project in this answer or proceed at step 2 with next team UUID

Is there any shorter and easieer way to get the team belonging to a given project?
The API call I'd dream of would be: /v1/acl/project/{uuid}.

Thank you for your help
Joerg

**Comment by stevespringett** [comment]
Related to https://github.com/CycloneDX/specification/issues/128

**Reply by JoergBruenner** [reply]
Hi @stevespringett,

thank you for your quick answer. I'm not shure if I understood https://github.com/CycloneDX/specification/issues/128 right.
It looks like bringing contact information to the SBoM. This wouldn't be my issue. I'd like to have an easier way to get 
some information already present in DT. I'm missing the reverse function of ​/v1​/acl​/team​/{uuid} call:
- ​API call: /v1​/acl​/project/{uuid}
- Input: Project UUID
- Output: List of teams assigned to the project

The team members I'd like to send a message I'll get from the /v1/team/{uuid} call.

Thank you for your help
Joerg
